src/OrganisationRegistry.Api/ApiModule.cs

```diff
@@ -98,12 +98,12 @@ private static Func<IServiceProvider> ProvideScopedServiceProvider(IComponentCon
         {
             var defaultServiceProvider = context.Resolve<IServiceProvider>();
 
-            if (!context.TryResolve(out IHttpContextAccessor httpContextAccessor))
+            if (!context.TryResolve(out IHttpContextAccessor? httpContextAccessor))
                 return () => defaultServiceProvider.CreateScope().ServiceProvider;
 
             return () =>
             {
-                var requestServices = httpContextAccessor?.HttpContext?.RequestServices;
+                var requestServices = httpContextAccessor.HttpContext?.RequestServices;
```

Comment:
is this correct? `IHttpContextAccessor` is nullable, but we don't use null-coalescing?

Reply:
Ok, TryResolve prevents this from happening.